website/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "scripts": {
     "start": "docusaurus start",
-    "build": "docusaurus build",
+    "build": "docusaurus build && yarn run update-redirect ./build/_redirects ./versions.json",
```

Comment:
Does this work for “docusaurus start”?

Reply:
Good question.  It doesn't work gracefully with `docusaurus start`.   I don't see a path to modify the webpack to support this.

I don't think this is blocking.